core/shared/utils/bh_time.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) 2023 Amazon Inc.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "bh_time.h"
+
+uint64
+convert_timespec(const struct timespec *ts)
```

Comment:
it'd be good to mention what we convert it to, e.g.
```suggestion
convert_timespec_to_nanoseconds(const struct timespec *ts)
```